app/Mage.php

```diff
@@ -176,25 +176,24 @@ public static function getVersionInfo()
      * Gets the current OpenMage version string
      * @link https://openmage.github.io/supported-versions.html
      * @link https://semver.org/
-     *
-     * @return string
      */
     public static function getOpenMageVersion(): string
     {
         $info = self::getOpenMageVersionInfo();
         $versionString = "{$info['major']}.{$info['minor']}.{$info['patch']}";
-        if ($info['stability'] || $info['number']) {
+        $stability = $info['stability'];
```

Comment:
We could use array destructuring here instead of two assignments 184-185. 
```php 
['stability' => $stability, 'number' => $number] = $info; 
```